docs/products/access/release-notes/1.x.md

```diff
@@ -1,5 +1,31 @@
 # Release notes 1.x versions
 
+## 1.1.1
+
+### Bug fixes
+
+* Resolved an issue causing some of the Access Admin's templates from being rendered correctly.
+
+## 1.1.0
+
+### Features
+
+* Mobile app developers can now modify the redirect URL that is configured in the Access Admin without breaking backwards compatibility with existing app installations.
+* Relying Party (RP) can resolve user attributes from the Access by calling the User Info API or requesting an ID-Token. Both means are defined by OpenID Connect (OIDC) standard. The returned set of identity related claims couldn't be modified, extended or filtered (other than by using scope). Introduced User Details Customization Web Hook that serve these purposes.
+* Whenever the user device domain state changes, which could be caused by a new user device registration, deregistration, user logging in with the device, or mobile authentication enrollment changes, Access will publish a corresponding event to the event bus notifying all interested parties about the change.   
+* The process of registering a new mobile application requires both parties, the device, and the server, to have their time/date settings set correctly. Some users are explicitly modifying their time which prevents them from successfully finishing the onboarding process. To improve the user experience Access will handle such situations more gracefully by detecting clock skew and informing the client about the root cause of the rejection.
+
+### Improvements
+
+* Extended scaling capabilities of the Access Admin service.
```

Comment:
I don't think we need to mention this because our customers won't scale the services. They will profit from this though :-)